api/client/lib.py

```diff
@@ -487,6 +487,9 @@ def list_of_series_to_single_series(series_list, add_belongs_to=False):
     for series in series_list:
         if not (isinstance(series, dict) and isinstance(series.get('data', []), list)):
             continue
+        series_metadata = series.get('series', {}).get('metadata', {})
+        if not include_historical and series_metadata.get('includesHistoricalRegion', False):
```

Comment:
There's also `'includesHistoricalPartnerRegion'`. Going to check for that as well?

Ex: `client.get_data_points(metric_id=11078, item_id=1, region_id=1079, partner_region_id=14, frequency_id=9, source_id=1)`

Also, I am noticing that the `'belongsTo'` series is including the metadata as well. Is that intended?

Reply:
this is intended. if the user requests data for a series selection, we expand to contained series, and i purposely mapped this back to the user selected series. see series-availability-utils addToAvailableSeriesSet  https://bitbucket.org/grointelligence/gro/src/3aa37b417e427aaa7b69dafbe70586a255e95585/api/lib/helpers/series-availability-utils.js#lines-368

Reply:
done. added check for historical partner, tested with above selections